### PR TITLE
k6 browser v1 migration guide

### PR DESCRIPTION
**Pull request by inancgumus**

### CODE REVIEW

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:74]: We don't need thresholds at this stage so we can remove this.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+</CodeGroup>
+
+
+
+
+
+## Import Path
+
+Changes have been made to the way the [browser module](/javascript-api/k6-experimental/browser/) is imported. With the browser type (i.e., `chromium`) now set in [scenario options](#scenario-options), users should directly import the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object from the [browser module](/javascript-api/k6-experimental/browser/). In the past, users would rely on `chromium.launch()` for accessing the running browser. Now, a simple import of the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) sufficient.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:99]:
```suggestion
Changes have been made to how the [browser module](/javascript-api/k6-experimental/browser/) is imported. With the browser type (i.e., `chromium`) now set in [scenario options](#scenario-options), users should directly import the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object from the [browser module](/javascript-api/k6-experimental/browser/). In the past, users relied on `chromium.launch()` for accessing the running browser. Now, a simple import of the [browser](/javascript-api/k6-experimental/browser/#browser-module-api) is sufficient.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        timeout: '60s',
+    });
+}
+```
+
+</CodeGroup>
+
+<Blockquote mod="note" title="">
+
+The following browser options are no longer supported: `slowMo`, `devtools`, `env`, and `proxy`.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:145]: Do we need a quick explanation here why they are no longer supported?

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+</Blockquote>
+
+
+
+
+## Scenario options
+
+In the new update, users must set the browser type as an option in a [k6 scenario](/using-k6/scenarios/) definition. Specifically, the `browser.type` option should be set to `chromium`, as Chromium is the only browser supported.
+
+This change helps identify the test as a browser test and allows automatic control of the browser's lifecycle. Users no longer need to start or stop the browser manually through the browser API. If the `browser.type` option is set in the scenario options, a browser instance will be automatically started at the beginning and closed at the end of each test iteration.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:156]:
```suggestion
This change helps identify the test as a browser test and allows automatic control of the browser's lifecycle. Users no longer need to start or stop the browser manually through the browser API. If the `browser.type` option is set in the scenario options, a browser instance will automatically start at the beginning and close at the end of each test iteration.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 5]
```diff
@@ -0,0 +1,249 @@
...
+    ui: {
+      executor: 'shared-iterations',
+      options: {
+        browser: {
+          type: 'chromium',
+        },
+      },
+    },
+  },
+  thresholds: {
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:174]: We can remove thresholds

**Review by mdcruz — COMMENTED** [review]
Looking great so far! I have a few comments :)

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 6]
```diff
@@ -0,0 +1,249 @@
...
+</CodeGroup>
+
+
+
+
+## Browser context limit
+
+Since the browser lifecycle is now being managed by the [browser module](/javascript-api/k6-experimental/browser/), the new browser implementation limits the use to a single active [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) per iteration. This change supports better prediction of resource requirements for a test run and allows for more controlled management of [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s.
+
+If a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) needs to be created, the existing one must be closed first using the [browserContext.close()](/javascript-api/k6-experimental/browser/browsercontext/close) method. And, a new one can be created either with the [browser.newContext()](/javascript-api/k6-experimental/browser/newcontext/) or [browser.newPage()](/javascript-api/k6-experimental/browser/newpage) methods.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:245]: Maybe we can show a code snippet here on how to create a new BrowserContext?

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 7]
```diff
@@ -0,0 +1,249 @@
...
+For all the details, check the complete changelog for the [k6 version 0.46](https://github.com/grafana/k6/releases/tag/v0.46.0).
+
+<Blockquote mod="note" title="">
+
+Users no longer need to use the `K6_BROWSER_ENABLED` flag when running browser tests with the `k6` command.
+
+</Blockquote>
+
+
+## Key Changes in the New API
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:18]:
```suggestion
## Key changes in the new API
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 8]
```diff
@@ -0,0 +1,249 @@
...
+}
+```
+
+</CodeGroup>
+
+
+
+
+
+## Import Path
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:97]:
```suggestion
## Import path
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 9]
```diff
@@ -0,0 +1,249 @@
...
+
+
+
+
+
+## Browser options
+
+In the updated version, the need to manually start a browser and set its configuration has been removed, and this part can simply be omitted from test scripts.
+
+Users can still change some browser settings by using environment variables. For more information, refer to the [browser module options](/javascript-api/k6-experimental/browser/#browser-module-options) documentation.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:125]: Would be useful to see an "After" code/test execution snippet on how to set `headless` and `timeout` by environment variables.

```bash
K6_BROWSER_HEADLESS=false K6_BROWSER_TIMEOUT='60s' k6 run script.js
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 10]
```diff
@@ -0,0 +1,249 @@
...
+The following browser options are no longer supported: `slowMo`, `devtools`, `env`, and `proxy`.
+
+</Blockquote>
+
+
+
+
+## Scenario options
+
+In the new update, users must set the browser type as an option in a [k6 scenario](/using-k6/scenarios/) definition. Specifically, the `browser.type` option should be set to `chromium`, as Chromium is the only browser supported.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:154]:
```suggestion
In the new update, users must set the [executor](/using-k6/scenarios/executors/) and browser type as options in a [k6 scenario](/using-k6/scenarios/) definition. Specifically, the `browser.type` option should be set to `chromium`, as Chromium is the only browser supported.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 9 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:125]: I omitted it because we already have an example in the link. Should we also duplicate one here? Or is it OK to provide a link as it is?

Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:125]: I omitted it because we have a link to the usage. Still it makes sense to before/after, though :)

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 5 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:174]: Agree 👍

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 3 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: 👍 Makes sense. Done.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 9 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:124]: Done 👍

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 6 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:269]: There are examples in the links. I'll write "check out the examples for creating a browsercontext".

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+```bash
+PS C:\k6> $env:K6_BROWSER_HEADLESS="false" ; $env:K6_BROWSER_TIMEOUT='60s' ; k6 run script.js
+```
+
+</CodeGroup>
+
+
+<Blockquote mod="note" title="">
+
+The following browser options are no longer supported: `slowMo`, `devtools`, `env`, and `proxy` as they were not proving much value to our users. In the meantime, we're working on providing a much flexible version of `slowMo` (slow motion) option.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:168]: I think `slowMo` was valuable, but the reason this was removed was because it's applicable to the browser level. Notes found here: https://docs.google.com/document/d/1fYtBc78F6TDek-BuXLnYSA-c_9pudq7w9OrMDXiL1jo/edit

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+
+Now, all that is needed is to specify the browser type within the [scenario options](#scenario-options). A browser instance will be automatically created and closed for each iteration by the browser module, streamlining the process.
+
+This change helps identify the test as a browser test and allows automatic control of the browser's lifecycle. Users no longer need to start or stop the browser manually through the browser API. If the `browser.type` option is set in the scenario options, a browser instance will automatically start at the beginning and close at the end of each test iteration.
+
+
+## Opening a new page
+
+You can open a new page by using the imported [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object's [browser.newPage()](/javascript-api/k6-experimental/browser/newpage) method. You can still using the [Page](/javascript-api/k6-experimental/browser/page/) object as before.
```
Comment (mdcruz) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:210]:
```suggestion
You can open a new page by using the imported [browser](/javascript-api/k6-experimental/browser/#browser-module-api) object's [browser.newPage()](/javascript-api/k6-experimental/browser/newpage) method. You can still use the [Page](/javascript-api/k6-experimental/browser/page/) object as before.
```

**Review by mdcruz — COMMENTED** [review]
LGTM! Just minor error and 1 comment :)

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+
+</CodeGroup>
+
+
+
+
+
+## Browser options
+
+In the updated version, the need to manually start a browser and set its configuration has been removed, and this part can simply be omitted from test scripts. Users can still change some browser settings by using environment variables. For more information, refer to the [browser module options](/javascript-api/k6-experimental/browser/#browser-module-options) documentation.
```
Comment (ka3de) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:122]: Small suggestion:
```suggestion
In k6 v0.46.0, the need to manually start a browser via `browser.launch()` or `browser.connect()` and set its configuration through these methods has been removed, so this part can simply be omitted from test scripts. Users can still change some browser settings by using environment variables. For more information, refer to the [browser module options](/javascript-api/k6-experimental/browser/#browser-module-options) documentation.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 14]
```diff
@@ -0,0 +1,274 @@
...
+The following browser options are no longer supported: `slowMo`, `devtools`, `env`, and `proxy` as they were not proving much value to our users. In the meantime, we're working on providing a much flexible version of `slowMo` (slow motion) option.
+
+</Blockquote>
+
+
+
+
+## Scenario options
+
+In the new version, users must set the [executor](/using-k6/scenarios/executors/) and browser type as options in a [k6 scenario](/using-k6/scenarios/) definition. Specifically, the `browser.type` option should be set to `chromium`, as Chromium is the only browser supported.
```
Comment (ka3de) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:177]:
```suggestion
In k6 v0.46.0, users must set the [executor](/using-k6/scenarios/executors/) and browser type as options in a [k6 scenario](/using-k6/scenarios/) definition. Specifically, the `browser.type` option should be set to `chromium`, as Chromium is the only browser supported.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 15]
```diff
@@ -0,0 +1,274 @@
...
+```
+
+</CodeGroup>
+
+
+
+
+## Browser context limit
+
+Since the browser lifecycle is now being managed by the [browser module](/javascript-api/k6-experimental/browser/), the new browser implementation limits the use to a single active [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) per iteration. This change supports better prediction of resource requirements for a test run and allows for more controlled management of [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s.
```
Comment (ka3de) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:266]: Should we mention here something about scenarios?. E.g.:
```suggestion
Since the browser lifecycle is now being managed by the [browser module](/javascript-api/k6-experimental/browser/), the new browser implementation limits the use to a single active [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) per iteration. This change supports better prediction of resource requirements for a test run, allows for more controlled management of [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/)s and promotes [scenarios](TODO:link) as the way to separate independent browser sessions.
```
cc: @mdcruz

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 16]
```diff
@@ -0,0 +1,274 @@
...
+
+Please click on the links below to see usage examples.
+
+* If a new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) needs to be created, the existing one must be closed first using the [browserContext.close()](/javascript-api/k6-experimental/browser/browsercontext/close) method.
+* A new [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/) can be created either with the [browser.newContext()](/javascript-api/k6-experimental/browser/newcontext/) or [browser.newPage()](/javascript-api/k6-experimental/browser/newpage) methods.
+* Alongside these changes, the method `browser.contexts()` has been altered to [browser.context()](/javascript-api/k6-experimental/browser/context/) to retrieve the current [BrowserContext](/javascript-api/k6-experimental/browser/browsercontext/).
+
```
Comment (ka3de) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:273]: Should we add a couple of tests to demonstrate what is allowed and what is not? E.g.: `new context -> close context -> new context` is allowed. But `new context -> new context` is not. ?

**Review by ka3de — COMMENTED** [review]
Thanks for this work @inancgumus , it's great! :clap: 
Just made a few small suggestions.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 17]
```diff
@@ -0,0 +1,274 @@
...
+title: 'Migrating to k6 v0.46'
+excerpt: 'A migration guide to ease the process of transitioning to the new k6 browser module version'
+slug: '/using-k6-browser/migrating-to-k6-v0-46/'
+---
+
+This guide outlines the key changes users will need to make when moving their existing k6 browser test scripts to the new browser module (bundled up with the k6 version 0.46). The updated version makes the browser module easier to use, with structural changes that reduce the need for manual setup, enhance scenario definitions, and make the code more straightforward.
+
+The new version presents a novel functionality that automatically deals with the browser lifecycle. Additionally, a newly required field is introduced to define the browser category within the scenario choices.
+
+Previously, users were responsible for establishing and shutting down the browser instance. However, with the current changes, the API has concealed the intricacies of `browser` and `browserType`, and the option to import the `chromium` entity from `k6/experimental/browser` has been eliminated.
```
Comment (ankur22) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:11]: I don't think we've abstracted away `browser`, since that is what users are now expected to work with, but we have abstracted away `browserType`. I've tried to reword this to reflect that change. WDYT?

```suggestion
Previously, users were responsible for establishing and shutting down the browser instance. However, with the current changes, the API has concealed the intricacies of `browserType`, and so the `chromium` named export has been removed from `k6/experimental/browser`, which has been replaced with `browser`.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 18]
```diff
@@ -0,0 +1,274 @@
...
+
+For all the details, make sure to review the complete changelog for the [k6 version 0.46](https://github.com/grafana/k6/releases/tag/v0.46.0) for all the information, watch [k6 Office Hours #98](https://www.youtube.com/watch?v=fK6Hpvt0pY0), where we discuss the latest changes in k6 browser, and, as always, ask in [the community forum](https://community.grafana.com/c/grafana-k6/k6-browser/79) if you need our help!
```
Comment (ankur22) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:13]:
```suggestion
For all the details, make sure to review the complete changelog for [k6 version 0.46](https://github.com/grafana/k6/releases/tag/v0.46.0). For more details watch [k6 Office Hours #98](https://www.youtube.com/watch?v=fK6Hpvt0pY0), where we discuss the latest changes in k6 browser, and, as always, ask in [the community forum](https://community.grafana.com/c/grafana-k6/k6-browser/79) if you need our help!
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 19]
```diff
@@ -0,0 +1,274 @@
...
+  } finally {
+    page.close();
+  }
+}
+```
+
+</CodeGroup>
+
+
+## Key changes in the new API
```
Comment (ankur22) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:83]:
```suggestion
## Key changes
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 11 above)
Comment (ankur22) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:168]:
```suggestion
The following browser options are no longer supported: `devtools`, `env`, and `proxy` since they weren't providing much value. `slowMo` has been removed for now, but we are working on adding it back in.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 20]
```diff
@@ -0,0 +1,274 @@
...
+        },
+      },
+    },
+  }
+}
+```
+
+</CodeGroup>
+
+Previously, users were required to handle the creation and closing of the browser instance using methods like `chromium.launch()` and `chromium.connect()` for creation, and the `browser.close()` method for releasing resources before ending the iteration. This repetitive code has been eliminated.
```
Comment (ankur22) [inline src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:201]:
```suggestion
Previously, users were required to handle the creation and closing of the browser instance using `chromium.launch()` or `chromium.connect()` for creation, and the `browser.close()` method for releasing resources before ending the iteration. This repetitive code has been eliminated.
```

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 11 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md:168]: I'll apply Ankur's [suggestion](https://github.com/grafana/k6-docs/pull/1292#discussion_r1293279128) here.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 15 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: For this, I've applied Ankur's suggestion.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 21]
```diff
@@ -0,0 +1,274 @@
+---
+title: 'Migrating to k6 v0.46'
+excerpt: 'A migration guide to ease the process of transitioning to the new k6 browser module version'
+slug: '/using-k6-browser/migrating-to-k6-v0-46/'
+---
+
+This guide outlines the key changes users will need to make when moving their existing k6 browser test scripts to the new browser module (bundled up with the k6 version 0.46). The updated version makes the browser module easier to use, with structural changes that reduce the need for manual setup, enhance scenario definitions, and make the code more straightforward.
```
[the remark this answers is not in the document]
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: I've applied the other suggestion that removes the second sentence.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 22]
```diff
@@ -0,0 +1,274 @@
...
+
+</CodeGroup>
+
+
+## Key changes in the new API
+
+The updated version introduces notable structural changes in its operation and API. Let's take a look at them.
+
+* [Scenario options](#scenario-options) must now be defined for running browser tests.
+* The [import path](#import-path) for the browser module has switched from `chromium` to [browser](/javascript-api/k6-experimental/browser/#browser-module-api).
```
[the remark this answers is not in the document]
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: 👍 Makes sense: 77d02592.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md [hunk 23]
```diff
@@ -0,0 +1,274 @@
...
+* Browser options can now only be set using certain [environment variables](/javascript-api/k6-experimental/browser/#browser-module-options). The `launch()` method, used earlier for this purpose, has been removed.
+* [Simplified resource management](#simplified-resource-management). The browser now starts automatically, managed by the browser module itself. There's no need to use `browser.close()` anymore.
```
[the remark this answers is not in the document]
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: Makes sense: 4b9a5036.

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: I've applied both of your suggestions 👍

File: src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md (hunk 6 above)
Comment (inancgumus) [reply src/data/markdown/translated-guides/en/03 Using k6 browser/04 Migrating to k6 v0-46.md]: Update: Provided an example.

**Review by mdcruz — APPROVED** [review]
Amazing. Thank you, @inancgumus! 🚀

**Review by ka3de — APPROVED** [review]
Great work @inancgumus ! :clap: 
LGTM.
